modules/auth/cms_user_temp_api.e

```diff
@@ -0,0 +1,127 @@
+note
+	description: "API to handle temporal users"
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_USER_TEMP_API
```

Comment:
it would be good to harmonize names for 

CMS_USER_TEMP_API and CMS_TEMPORAL_USER

I would go for

CMS_TEMP_USER_API 
and CMS_TEMP_USER

And even if "temporal" is a valid term, I think "temporary" would be better.

- temporal means "of or relating to time as opposed to eternity"
- temporary means "lasting for a limited time"

Now I may be wrong as well...